fizz/CMakeLists.txt

```diff
@@ -237,6 +240,7 @@ target_include_directories(
     ${GLOG_INCLUDE_DIRS}
     ${FIZZ_INCLUDE_DIRECTORIES}
     ${DOUBLE_CONVERSION_INCLUDE_DIRS}
+    ${ZLIB_INCLUDE_DIRS}
```

Comment:
You should  modify   `target_link_libraries(fizz PUBLIC ZLIB::ZLIB)` (the cmake target) which will automatically add the correct headers to the build. You do not need to use `target_include_directories`.

zlib is a public dependency (not private), since `fizz/protocol/ZlibCertificateCompressor.h` includes `zlib.h`.